src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -214,7 +214,8 @@
       queryable: true,
       visibleByDefault: true,
       essential: true,
-      valueGetter: ":sourceAlias.jsonb->>'source'",
```

Comment:
Do we still need the sourceAlias prop, then?

Reply:
Nope. Only needed if it's used